engine/src/main/scala/cromwell/engine/workflow/lifecycle/execution/job/EngineJobExecutionActor.scala

```diff
@@ -717,6 +719,8 @@ class EngineJobExecutionActor(replyTo: ActorRef,
     } else {
       workflowLogger.info("Call cache hit process had 0 total hit failures before completing successfully")
     }
+
+    publishCopyAttemptFailuresMetrics(data)
```

Comment:
We publish copy attempt failures metrics even if there were no copy attempt failures, but not similarly for abandonments?

Reply:
It was previously being called next to `publishCopyAttemptAbandonedMetric()`, but I've moved it inside that function in case we ever call the same function from somewhere else.